presto-parquet/pom.xml

```diff
@@ -238,6 +238,11 @@
                         <artifactId>fmpp</artifactId>
                         <version>0.9.15</version>
                     </dependency>
+                    <dependency>
+                        <groupId>junit</groupId>
+                        <artifactId>junit</artifactId>
+                        <version>4.13.2</version>
```

Comment:
@anilsomisetty from `mvn dependency:tree`, I don't see `JUnit` as a dependency.  I believe it's because it's test-scoped in `fmpp-maven-plugin`.  If it's not being brought in, why do we need to do this?

Reply:
Yes it's a test scoped dependency and we don't need this, will close this PR, Thanks.